64_allowlist_signing.md

```diff
@@ -0,0 +1,290 @@
+- `--transparency-log-url`: to provide the server address of a compatible transparency log. Defaults to `rekor.sigstore.dev`
+
+In terms of specific implementation:
+
+- When loading an allowlist from disk, if the `--allowlist-verify-transparency-log` flag is set, the tenant will check the transparency log specified by `--transparency-log-url` for an entry that matches the provided artifacts.
+- Then, it will verify the inclusion proof using the results returned from the transparency log. If valid, allowlist loading continues; if not, it fails with an error.
+
+### Test Plan
+
+<!--
+**Note:** *Not required until targeted at a release.*
+
+Consider the following in developing a test plan for this enhancement:
+- Will there be e2e and integration tests, in addition to unit tests?
+- How will it be tested in isolation vs with other components?
+
+No need to outline all of the test cases, just the general strategy.  Anything
+that would count as tricky in the implementation and anything particularly
+challenging to test should be called out.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations).
+-->
+
+- A new test should be written to ensure that the allowlist loading process checks for inclusion proof in a transparency log.
+
+
+### Upgrade / Downgrade Strategy
+
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this enhancement
+-->
+
+Since this feature will take the form of a new option in Keylime’s tenant command, there should be no adverse impacts to upgrading or downgrading.
+
+### Dependency requirements
+
+<!--
+If your new change requires new dependencies, please outline and demonstrate that your selected dependency
+is well maintained and packaged in Keylime's supported Operating Systems (currently Debian Stable
+and as of time writing Fedora 32/33).
+
+During code implementation you will also be expected to add the package to CI , the keylime ansible role and
+keylimes main installer (`keylime/installers.sh`).
+
+If the package is not available in the supported Operated systems, the PR will not be merged into master.
+
+Adding the package in `requirements.txt` is not sufficent for master which is where we tag releases from.
+
+You may however be able to work within an experimental branch until a package is made available. If this is
+the case, please outline it in this enhancement.
+
+-->
+
+The cryptography and web interaction required involve the use of several Python libraries, most notably the `cryptography.*` library, though these should be available through Python’s package system if not installed already.
```

Comment:
We are already using the `cryptography` package, so you can note that no new dependencies will be added.